.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,16 +7,18 @@ assignees: ''
 
 ---
 
-[Give a short summary of what the problem is. Note: Before submitting an error report, please test if the problem is reproducible in the latest Slicer Preview Release. If the problem does not occur in the latest Slicer Preview Release then most likely it has been already fixed and there is no need to report it again.]
+## Summary
+
+[ Give a short summary of what the problem is. Note: Before submitting an error report, please test if the problem is reproducible in the latest Slicer Preview Release. If the problem does not occur in the latest Slicer Preview Release then most likely it has been already fixed and there is no need to report it again. If you are not sure if Slicer works incorrectly or you are not using it as intended then clarify it by asking on the Slicer Forum - https://discourse.slicer.org ]
```

Comment:
Should we write these comments using

```
<!-- Give a short summary of what the  ... https://discourse.slicer.org  -->
```

instead of 

```
[[ Give a short summary of what the  ... https://discourse.slicer.org  ]]
```

That way these wouldn't be posted as part of the issue description.

Reply:
Yes, that is a good idea. I remember seeing it done in ITK’s template. https://github.com/InsightSoftwareConsortium/ITK/blob/master/.github/ISSUE_TEMPLATE/regression_bug_report.md

Reply:
Agree, it's a good idea. I'll also change this in the ExtensionsIndex pull request template, where developers notoriously leave the instructions in the code.